src/uct/ib/base/ib_iface.c

```diff
@@ -17,6 +17,8 @@
 #include <poll.h>
 
 
+#define UCT_IB_DEVICE_INFO_TABLE_SZ 5
```

Comment:
would be better to mark the last entry as {0,NULL,0,0} - the when we add new entry, would have to change only one place